.pre-commit-config.yaml

```diff
@@ -35,12 +35,12 @@ repos:
       - id: pyupgrade
         args: [--py311-plus]
 
-  - repo: https://github.com/myint/docformatter
-    rev: v1.7.5
-    hooks:
-      - id: docformatter
-        args: [--in-place, --wrap-summaries=120, --wrap-descriptions=120]
-        exclude: '^.*__init__\.py$'
+  # - repo: https://github.com/myint/docformatter
```

Comment:
@JeanMaximilienCadic I commented this to make the build pass on my machine a few days ago and it worked. Now I am getting a new error as described in my last comment.